apps/api/src/main/java/com/agricloud/service/InfoService.java

```diff
@@ -0,0 +1,36 @@
+package com.agricloud.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.agricloud.model.RegionModel;
+import com.agricloud.repository.InfoRepository;
+import com.agricloud.response.GeneralResponse;
+
+@Service
+public class InfoService {
+    
+    @Autowired
+    private InfoRepository infoRepository;
+
+    public GeneralResponse getRegions() {
+        
+        GeneralResponse response = new GeneralResponse();
+
+        try {
+                
+            Iterable<RegionModel> regions = infoRepository.findAll();
+            response.setStatus("Successfully retrieved regions");
+            response.setBody(regions);
```

Comment:
Somewhere in the setup, this is clearing the mock data!

But, good news is, it is properly returning the information of the data if I re-add it while running.